### PR TITLE
✨📖 Custom desktop one panel aspect ratio - validator and documentation

### PR DESCRIPTION
**Pull request by ychsieh**
#38658

### CODE REVIEW

File: extensions/amp-story/amp-story.md [hunk 1]
```diff
@@ -548,6 +548,10 @@ Disables the [Live story](#Live-story) functionality.
 
 Used with the `live-story` attribute. Time interval (in milliseconds) between checks for new content. If no `data-poll-interval` is provided it with default to the 15000 millisecond minimum. A value under 15000 milliseconds is invalid.
 
+### desktop-aspect-ratio (optional)
+
+The value specifies an aspect ratio in the "horizontal:vertical" format, where both "horizontal" and "vertical" are integer numbers. If this attribute is specified, the layout of the story in desktop one panel mode is set to conform to the specified proportions.
```
Comment (processprocess) [inline extensions/amp-story/amp-story.md:553]: nit: Should we mention the min and max ratios?

**Review by processprocess — APPROVED** [review]
This looks good to me but I'll defer to @mszylkowski or @gmajoulet for final approval.

**Review by mszylkowski — APPROVED** [review]

File: extensions/amp-story/amp-story.md (hunk 1 above)
Comment (ychsieh) [reply extensions/amp-story/amp-story.md]: Done.

**Review by erwinmombay — APPROVED** [review]

**Review by amaltas — APPROVED** [review]

**Comment by edoudou** [comment]
Any idea when validation will be in stable ?

**Comment by edoudou** [comment]
This is marked as stable but still not active in the validator : https://validator.ampproject.org/#url=https%3A%2F%2Fjoin-dev.my.join-stories.com%2Fstory-1941%2F%3Ft%3D051

Any idea when it will be ?

**Comment by ychsieh** [comment]
Thanks for following up. It is active in the validator now. Do you mind share an example page that uses this custom desktop aspect ratio?

**Pull request merged**